### ISSUE TITLE
why attention mask is plus, should it be multiply attention mask?

### ISSUE THREAD

**Issue by joey-wang123** [microsoft/Oscar#24]
Hello,

Thanks for your interesting work. I recently read this code, some part I did not fully understand. Could you explain the attention mask why is added to the original attention score? In this case, the attention can still attend to future token, right?  Thanks!
https://github.com/microsoft/Oscar/blob/a9013bb7dda35a63856d1cebd16eeeeb73615e5c/oscar/modeling/modeling_bert.py#L48

**Comment by xiyinmsu** [comment]
The attention_mask here is not a 0/1 mask. You can check this: https://github.com/microsoft/Oscar/blob/a9013bb7dda35a63856d1cebd16eeeeb73615e5c/oscar/modeling/modeling_bert.py#L223
